build.sbt

```diff
@@ -291,7 +291,10 @@ lazy val taskProj = (project in file("tasks"))
     name := "Tasks",
     mimaSettings,
     mimaBinaryIssueFilters ++= Seq(
-      exclude[ReversedMissingMethodProblem]("sbt.ExecuteProgress.stop")
+      // ok because sbt.ExecuteProgress has been under private[sbt]
```

Comment:
Hopefully the community build passes. It wouldn't shock me if there are some plugins that use it given how far back it goes, but obviously a plugin author uses private[sbt] apis at their own risk.